packages/at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -407,6 +410,11 @@ class AtLookupImpl implements AtLookUp {
     return await _process(atCommand, auth: true);
   }
 
+  Future<String> _enroll(EnrollVerbBuilder builder) async {
```

Comment:
need unit or functional test to verify this method

Reply:
Unit tests added in [bf70ac7](https://github.com/atsign-foundation/at_libraries/pull/657/commits/bf70ac75fa3b22cd2f9dc2ac0e8a64cfbd63a9bb)